packages/ui/src/ReservoirKitProvider.tsx

```diff
@@ -31,6 +31,7 @@ type ReservoirKitProviderOptions = {
   disableJumperLink?: boolean
   coinGecko?: CoinGecko
   alwaysIncludeListingCurrency?: boolean
+  switchMainCurrency?: boolean
```

Comment:
Don't love this naming, it should be a little more descriptive of what's happening. Something like: `preferDisplayFiatTotal`